### PR TITLE
Reorganise repo and add linting

### PR DESCRIPTION
**Pull request by WolfspiritM**
# Outstanding tasks

- [x] Fix linting errors #3 
- [ ] Add basic tests #2 
- [x] Integrate with Circle CI #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-# chroma.js
+# Chroma.js
+
+[![CircleCI](https://circleci.com/gh/chroma-sdk/chroma-js.svg?style=svg&circle-token=d066d70fae54ca26b07aad36250e78d478c62790)](https://circleci.com/gh/chroma-sdk/chroma-js)
+
+Chroma.js is a library that provides a simple interface to interact with Razer's REST api.
+
+## Getting started
```
Comment (njbmartin) [inline README.md]: Change to "Building from source"

Comment (njbmartin) [reply README.md]: Once up on the NPM registry, we'll add a getting started section.

File: src/Animation.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import {AnimationFrame} from "./AnimationFrame";
+import {ChromaInstance} from "./ChromaInstance";
+import Color from "./Color";
+import {DeviceRequestData} from "./DeviceRequestData";
+import DeviceContainer from "./Devices";
+import { IDevice, IDeviceData } from "./Devices/Base";
```
Comment (njbmartin) [inline src/Animation.ts]: Are we sure the linting works? Looks like a mixture of formatting...

Comment (WolfspiritM) [reply src/Animation.ts]: Yes the linting works. :-)
I even made sure that it's called for all subfolders as it didn't before.

It's two different types of import depending if the element imported is a default export or not.
We might however think about not using default exports in general to make it more consistent.
Linting does only care about order of elements. So reordering this to have imports and default imports grouped will cause a linting error cause names aren't ordered alphabetically.

**Review by njbmartin — DISMISSED** [review]
Great work! Let's get this up on master before we add tests.

**Review by njbmartin — APPROVED** [review]
Let's merge this!

**Comment by njbmartin** [comment]
Closes #3 and #4

**Pull request merged**